src/pages/Guilds/Guilds.tsx

```diff
@@ -47,20 +49,31 @@ const proposalsMock = [
 ];
 
 const GuildsPage: React.FC = () => {
+  // @TODO: parse guild contract address from where?
+  // @TODO: loading designs
+  // @TODO: error designs
+  const GUILDS_ADDRESS = '0x9cDC16b5f95229b856cBA5F38095fD8E00f8edeF';

```

Comment:
Issue mentions pulling address from the URL

> Add guild address to the url
#464

Reply:
https://github.com/DXgovernance/dxvote/pull/478#discussion_r769497493